ambari-agent/conf/unix/ambari-agent.service

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific
+
+# This service unit file is tuned for SLES 12.x. It is not suitable for RHEL like distributions due to different
+# paths for ExecStart/ExecStop.
+
+[Unit]
+Description=Ambari Agent Service for SLES
+Before=runlevel3.target
+Before=runlevel5.target
+Before=shutdown.target
+After=remote-fs.target
+After=network-online.target
+Wants=network-online.target
+Conflicts=shutdown.target
+
+[Service]
+Type=forking
+Restart=no
```

Comment:
are you sure about this? currently CB deploys it with always

Reply:
I would like to keep consistency with other systems and make the minimal change needed.